### PR TITLE
[doc] Update README.md

### PR DESCRIPTION
**Pull request by pengzhendong**

### CODE REVIEW

File: runtime/device/android/wenet/README.md [hunk 1]
```diff
@@ -25,3 +25,31 @@ then just build and run the APK. Here is a gif demo, which shows how our on-devi
 Please note the wifi and data has been disabled in the demo so there is no network connection ^\_^.
 
 ![Runtime android demo](../../../../docs/images/runtime_android.gif)
+
+## Compute the RTF
+
+You could build the binary and the APK with Android Studio directly, or with the commands as follows:
```
Comment (robin1001) [inline runtime/device/android/wenet/README.md:31]: It's better we make the logic explicit by First, Second, Third, or Step 1, Step 2, Step 3

Comment (robin1001) [reply runtime/device/android/wenet/README.md:31]: Maybe step 1, step 2, step 3 is better

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline

File: runtime/device/android/wenet/README.md (hunk 1 above)
Comment (pengzhendong) [reply runtime/device/android/wenet/README.md]: Done.

**Review by robin1001 — APPROVED** [review]

**Pull request merged**